src/renderer/components/avatar/avatar.tsx

```diff
@@ -81,7 +74,11 @@ export function Avatar(props: AvatarProps) {
         [styles.rounded]: variant == "rounded",
         [styles.disabled]: disabled,
       }, className)}
-      style={{ width: `${size}px`, height: `${size}px`, backgroundColor: getBackgroundColor() }}
+      style={{
+        width: `${size}px`,
+        height: `${size}px`,
```

Comment:
Pixels are default numeric values in React's inline style prop.

```suggestion
        width: size,
        height: size,
```

Reply:
Didn't know that. Thanks.